Makefile

```diff
@@ -110,7 +110,9 @@ test-go: libflux-go
 	$(GO_TEST) $(GO_TEST_FLAGS) ./...
 
 test-rust:
-	cd libflux && $(CARGO) test $(CARGO_ARGS) && $(CARGO) clippy $(CARGO_ARGS) -- -Dclippy::all
+	cd libflux && $(CARGO) test $(CARGO_ARGS) && \
+	$(CARGO) test --doc && \
```

Comment:
From the [cargo book](https://doc.rust-lang.org/cargo/commands/cargo-test.html):

> --doc
> Test only the library's documentation. This cannot be mixed with other target options.